app/views/cars/index.html.slim

```diff
@@ -9,3 +9,6 @@ h1 Cars
       = link_to "Show this car", car
 
 = link_to "Add a new car", new_car_path
+br
+br
+= link_to "Back", root_path
```

Comment:
But cars#index will be our root if it isn't already. In that case this will just reload the page, so just delete it

Reply:
You don't need these too just pull main

Reply:
I put them for easier use it's not something special, but I will change it.